lib/tasks/offences.rake

```diff
@@ -18,4 +21,27 @@ namespace :offences do
     repair = Tasks::RakeHelpers::RepairOffences.new(file: args.filename, output: args.output, interactive:, dry_run:)
     repair.repair
   end
+
+  desc 'Extract details of offences and write to a CSV file'
+  task :extract, [:dir] => :environment do |_task, args|
+    dir = args.dir
+    if File.exist?(dir)
```

Comment:
I guess if I were being nitpicky, it might be worth specifying a default directory in case the user doesn't provide one themselves, otherwise this line will fail with an error. But as this is mostly just for our use in this specific case, it's probably overkill.